wiki/markdown/curl-examples.md

```diff
@@ -273,23 +273,24 @@ The response is a json object containing all the connected components and data a
         <th>Explanation</th>
     </tr>
     <tr>
-        <td>/queryAggregatedObject</td>
+        <td>/aggregated-objects/{id}</td>
         <td>GET</td>
-        <td>Download a subscription template json file</td>
+        <td>query an aggregated object with its id</td>
+    </tr> 
+    <tr>
+        <td>/aggregated-objects/query</td>
+        <td>POST</td>
+        <td>Free style query; user specified queries</td>
     </tr>
     <tr>
-        <td>/queryMissedNotifications</td>
+        <td>/failed-notifications</td>
```

Comment:
This should probably be moved to its own header now since they are separate endpoints.

Reply:
updating in the next commit